docs/docs/authenticate/oidc/auth0.mdx

```diff
@@ -1,8 +1,14 @@
 ---
 title: Auth0
 sidebar_position: 1
+description:
+    Enforce 2FA/MFA for users of Firezone's WireGuard®-based
+    secure access platform. This guide walks through integrating Auth0
+    for single sign-on using OpenID Connect (OIDC).
```

Comment:
Is this valid YAML? I think you need the `>` or `|` multiline indicator.

Reply:
Seems like it's valid. It shows up in the header correctly: 
<img width="403" alt="image" src="https://user-images.githubusercontent.com/52545545/206876840-cc29b476-334d-415c-b4a7-89d6dbd4572d.png">